controller.go

```diff
@@ -18,9 +18,9 @@ package main
 import (
 	"flag"
 	"fmt"
-	"github.com/redhatinsighs/insights-operator-controller/logging"
```

Comment:
You missed typo - "RedHatInsights" =)

Reply:
Sorry, It was bulk typo. Now fixed.

Reply:
I thought it would make more clear that its dockerfile for postgres and not for the controller, or sqllite. Should I still rename it ?

Reply:
No, I think it is not necessary.